core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -27,50 +27,60 @@
 {% endblock header_js %}
 
 {% block content %}
-  <div ng-controller="CollectionEditor">
-    <div ng-if="collection">
-      <!-- Collection details -->
-      <collection-details-editor-directive collection="collection">
-      </collection-details-editor-directive>
-      <a ng-href="/collection/<[collection.getId()]>" target="_blank">Go to Collection</a>
+  <div class="oppia-page-cards-container">
```

Comment:
As discussed offline, we're going to do this in #1748 instead to avoid extra conflicts.